demos/mobile/android/app/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
Everything is commented out here.  Is that intentional?

Reply:
Yes.  That is the default boilerplate from Android Studio